templates/membership/contracts/MembershipTemplate.sol

```diff
@@ -0,0 +1,98 @@
+pragma solidity 0.4.24;
+
+import "@aragon/templates-shared/contracts/BaseTemplate.sol";
+
+
+contract MembershipTemplate is BaseTemplate {
+    string private constant ERROR_MISSING_TOKEN_CACHE = "MEMBERSHIP_MISSING_TOKEN_CACHE";
+    string private constant ERROR_MISSING_MEMBERS = "MEMBERSHIP_MISSING_MEMBERS";
+
+    bool constant private TOKEN_TRANSFERABLE = false;
+    string constant private TOKEN_NAME = "Member Token";
+    string constant private TOKEN_SYMBOL = "MEMBER";
+    uint256 constant private TOKEN_MAX_PER_ACCOUNT = uint256(1e18);
```

Comment:
I think we should just use `1` instead, and leave the token without any decimals, like before.